src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/SealedClaimPdfTest.java

```diff
@@ -68,9 +69,14 @@ private String getFullAddressString(Address address) {
     }
 
     private Claim createCase() {
+        UUID externalId = UUID.randomUUID();
```

Comment:
No need to create it manually, `testData.submittedByClaimantBuilder()` is generating unique UUID, just do 

```
ommonOperations.submitPrePaymentClaim(claimData.getExternalId().toString(), claimant.getAuthorisation());
```

Reply:
done